File: Webshop2/Webshop2/src/DTO/UserDTO.java
package DTO;

public class UserDTO {


	private String firstname;
	private String lastname;

	public UserDTO(String firstname, String lastname) {

	
		this.firstname = firstname;
		this.lastname = lastname;

	}








	public String getFirstname() {

		return this.firstname;

	}

	public void setFirstname(String firstname) {

		this.firstname = firstname;

	}
	
	
	public String getLastname() {
		
		return this.lastname; 
		
		
	}
	
	
	public void setLastname(String lastname) {
		this.lastname = lastname;
	}
	

}

File: C#/Eerdere versie/WebshopC#Con5NA BUG FIX/WebshopC#Con5/Infrastructure/DTO/UserDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Infrastructure.DTO
{
    public class UserDTO
    {

        public string Username { get; set; }
        public string Email { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }


        public UserDTO() { }
        

        public UserDTO(string username, string email, string firstname, string lastname)
        {
            Username = username;
            Email = email;
            Firstname = firstname;
            Lastname = lastname;

        }


    }
}
